Sources/xcodeproj/Extensions/Dictionary+Enumerate.swift

```diff
@@ -7,11 +7,15 @@ extension Dictionary {
         using block: (Any, Any, UnsafeMutablePointer<ObjCBool>) throws -> Void) throws
     {
         var blockError: Error?
-        (self as NSDictionary).enumerateKeysAndObjects(options: opts) { (key, obj, stops) in
+        // For performance it is very important to create a separate dictionary instance.
+        // (self as NSDictionary).enumerateKeys... - works much slower
+        let dictionary = NSDictionary(dictionary: self)
```

Comment:
@thedavidharris  Should I add `#if os(macOS)` ? + else implementation

Reply:
It's just the `ObjCBool` here that doesn't port, if there's an implementation of this that doesn't use that then it's fine.

Reply:
It's just the ObjCBool here that doesn't port, if there's an implementation of this that doesn't use it, that would be best